### PR TITLE
feat: bff 653/discover

### PR DESCRIPTION
**Pull request by alexbejann**
## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/features/give/widgets/choose_category.dart [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                child: FilterSuggestionCard(
+                  isFocused: false,
+                  color: CollecGroupType.charities.color,
+                  title: _getCategoryName(
+                    type: CollecGroupType.charities,
+                    context: context,
+                  ),
+                  activeIcon: CollecGroupType.charities.activeIcon,
+                  icon: CollecGroupType.charities.icon,
+                  width: size.width * 0.4,
```
Comment (MaikelStuivenberg) [inline lib/features/give/widgets/choose_category.dart:68]: Not sure if this was existing code, but we should make use of the flex layouts (*column, row, expanded, margin, etc). This is *0.3/0.4 for width, heights are not scalable enough for all screen sizes

File: lib/features/give/widgets/filter_suggestion_card.dart [hunk 2]
```diff
@@ -8,70 +8,79 @@ class FilterSuggestionCard extends StatelessWidget {
...
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Container(
+                    margin: const EdgeInsets.symmetric(
+                      vertical: 5,
+                      horizontal: 5,
+                    ),
+                    child: Image.asset(
+                      isFocused ? activeIcon : icon,
+                      width: size.width * (isFocused ? 0.08 : 0.06),
```
Comment (MaikelStuivenberg) [inline lib/features/give/widgets/filter_suggestion_card.dart:72]: Also this is a bit ugly, I guess we can fix that differently and more scalable

Comment (alexbejann) [reply lib/features/give/widgets/filter_suggestion_card.dart:72]: Yep, it was existing. I gotta change it

**Review by MaikelStuivenberg — APPROVED** [review]

File: lib/features/give/widgets/choose_category.dart (hunk 1 above)
Comment (alexbejann) [reply lib/features/give/widgets/choose_category.dart:68]: Just tested on an iPhone SE and it's looks ok. But indeed I would have to change it in the end

**Pull request merged**